Challenge03/html/flor.html

```diff
@@ -0,0 +1,181 @@
+            has roots in blues rock, psychedelic rock and acid rock, but also
+            incorporates elements of classical music, punk rock, hip-hop and
+            other genres. Some of the most famous pioneers of metal music are
+            Led Zeppelin, Black Sabbath and Deep Purple, while some of the most
+            popular and influential bands are Metallica, Iron Maiden, Judas
+            Priest and Slayer. Metal music has many subgenres that vary in
+            style, speed, aggression and themes, such as heavy metal, thrash
+            metal, death metal, black metal, power metal, symphonic metal and nu metal. Metal music is often associated with a subculture of fans
+            called "metalheads" or "headbangers", who enjoy the loudness, energy
+            and complexity of the music.
+        </p>
+      </section>
+      <section  id="subgenres" class="main__section section">
+        <h2 class="section__title">Subgenres</h2>
+<a href="#history" class="nav__link">+ Learn more</a>
+
+        <p class="section__text">
+            Metal music is a genre of rock music that has evolved and
+            diversified over the decades, creating a wide range of subgenres
+            that appeal to different tastes and audiences. Metal music is known
+            for its loudness, energy, complexity and creativity, as well as its
+            controversial and rebellious themes. Metal music has influenced and
+            been influenced by many other genres of music, such as classical,
+            punk, hip-hop and electronic. Metal music has also spawned a loyal
+            and passionate fanbase, who identify themselves as metalheads or
+            headbangers. Metal music is more than just a genre, it is a culture
+            and a lifestyle for many people around the world. Some subgenres of 
+            metal music are: Heavy Metal, Power Metal, Thrash Metal, Death Metal and Symphonic Metal.
+        </p>
+      </section>
+      <section id="moreAbout" class="main__section section">
+        <h2 class="section__title">Culture</h2>
+        <p class="section__text">
+            Metal music is a genre of rock music that has evolved and
+            diversified over the decades, creating a wide range of subgenres
+            that appeal to different tastes and audiences. Metal music is known
+            for its loudness, energy, complexity and creativity, as well as its
+            controversial and rebellious themes. Metal music has influenced and
+            been influenced by many other genres of music, such as classical,
+            punk, hip-hop and electronic. Metal music has also spawned a loyal
+            and passionate fanbase, who identify themselves as metalheads or
+            headbangers. Metal music is more than just a genre, it is a culture
+            and a lifestyle for many people around the world.
+        </p>
+      </section>
+    </main>
+   <hr>
+  <div id="subgenres" class="subgenres">

```

Comment:
The id attribute must be unique within the HTML document. In your code, both the <section> element with id="subgenres" and this <div> element with id="subgenres" have the same id value. You should modify one of them to have a different id.